build/toolchain/toolchain.gni

```diff
@@ -49,15 +49,15 @@ assert(is_starboard || !use_xcode_clang || target_os == "ios",
 
 # Extension for shared library files (including leading dot).
 if (is_apple) {
-  shlib_extension = ".dylib"
+  shared_lib_extension = ".dylib"
```

Comment:
Why are we renaming this?

Reply:
because of https://b.corp.google.com/issues/246412559#comment79

Reply:
resolved offline - took alternative approach